index.js

```diff
@@ -5,6 +5,12 @@ const baseUrl = 'https://min-api.cryptocompare.com/data/'
 
 function fetchJSON (url) {
   return fetch(url)
+    .then(res => {
+      if (!res.ok) {
+        throw Error(res.status + ' ' + res.statusText)
```

Comment:
Use `new` when creating an Error, and use template strings instead of `+`.
```js
throw new Error(`${res.status} ${res.statusText}`)
```

Reply:
Ok, no problem.